test/builder-api/src/ext.js

```diff
@@ -7,23 +7,6 @@ const installationId = 56940;
 const repoId = 114932712;
 
 describe('External API', function() {
-  describe('Searching github code', function() {
-    it('requires authentication', function(done) {
-      request.get(`/ext/installations/${installationId}/search/code`)
-        .accept('application/json')
-        .expect(401)
-        .end(function(err, res) {
-          expect(res.text).to.be.empty;
-          done(err);
-        });
-    });
-
-    // This one requires correct GH creds to test. Authenticating with legit GH
-    // creds would break the janky session short-circuiting we have for these
-    // tests. A bit of a catch-22.
-    it('succeeds');
```

Comment:
Not gonna lie - I'm pretty stoked to see this deleted 😁

Reply:
I bet--a call to an external service in a test? One of the hardest things!